app/views/shared/_assistant_sidebar_link.html.erb

```diff
@@ -34,12 +35,11 @@
       </div>
     <% end %>
 
-    <%= link_to new_assistant_message_path(assistant) do %>
+    <%= link_to new_assistant_message_path(assistant), class: "inline-flex" do %>
       <%= icon "pencil-square",
         variant: :micro,
         size: 18,
-        class: "inline text-gray-400 cursor-pointer group-hover:visible hover:text-white #{selected ? 'text-white visible' : 'invisible'}",
-        data: { selected: selected },
+        class: "text-gray-400 cursor-pointer group-hover:visible hover:text-white #{selected ? 'text-white visible' : 'invisible'}",
```

Comment:
In the ~10 lines above I tweaked the placement of "inline" because it was preventing the icon from slightly depressing and shrinking in size upon click.